probe-dictionary/explorer.css

```diff
@@ -155,3 +155,7 @@ th {
     top: -1px;
     background-color: #ECECEC;
 }
+
+#detail-probe-name {
+    overflow-wrap: break-word;
+}
```

Comment:
A very small thing: This is missing a linebreak. Usually we have an empty newline at the end of files, see [e.g. here](https://stackoverflow.com/a/729795/).

Reply:
Yes done!

Reply:
Thank you!